modules/browser/BrowserCapabilities.js

```diff
@@ -104,10 +104,21 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean} true if the browser is supported, false otherwise.
      */
     isSupported() {
-        return this.isChromiumBased()
-            || this.isFirefox()
-            || this.isReactNative()
-            || this.isSafariWithWebrtc();
+        const REQUIRED_CHROME_VERSION = 69;
+
+        if (this.isChrome()) {
```

Comment:
why do we check for Chrome excplicitly, if it's already part of the  this.isChromiumBased() check?

Reply:
Yep we don't need the chrome check.